spec/models/changeset_spec.rb

```diff
@@ -30,14 +30,14 @@
 
 describe Changeset, :type => :model do
   let(:email) { "bob@bobbit.org" }
-  let(:repo) { FactoryGirl.create(:repository) }
-  let(:changeset) { FactoryGirl.build(:changeset,
-                                      repository: repo,
-                                      revision: '1',
-                                      committer: email,
-                                      comments: "Initial commit") }
-
-  before { allow(Setting).to receive(:enabled_scm).and_return(['Filesystem']) }
+
+  with_created_filesystem_repository do
+    let(:changeset) { FactoryGirl.build(:changeset,
+                                        repository: repository,
+                                        revision: '1',
+                                        committer: email,
+                                        comments: "Initial commit") }
```

Comment:
Expression at 39, 69 should be on its own line.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.